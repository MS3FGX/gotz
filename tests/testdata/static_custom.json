{
  "config_version": "1.0",
  "timezones": [
    {
      "Name": "New York",
      "TZ": "America/New_York"
    },
    {
      "Name": "Berlin",
      "TZ": "Europe/Berlin"
    },
    {
      "Name": "Shanghai",
      "TZ": "Asia/Shanghai"
    },
    {
      "Name": "Sydney",
      "TZ": "Australia/Sydney"
    }
  ],
  "style": {
    "symbols": "custom",
    "custom_symbols": [
      {
        "Start": 6,
        "Symbol": "O"
      },
      {
        "Start": 8,
        "Symbol": "X"
      },
      {
        "Start": 18,
        "Symbol": "O"
      },
      {
        "Start": 22,
        "Symbol": "_"
      }
    ],
    "colorize": true,
    "day_segments": {
      "morning": 6,
      "day": 8,
      "evening": 18,
      "night": 22
    },
    "coloring": {
      "StaticColorMorning": "red",
      "StaticColorDay": "yellow",
      "StaticColorEvening": "red",
      "StaticColorNight": "blue",
      "StaticColorForeground": "",
      "DynamicColorMorning": "red",
      "DynamicColorDay": "yellow",
      "DynamicColorEvening": "red",
      "DynamicColorNight": "blue",
      "DynamicColorForeground": "",
      "DynamicColorBackground": ""
    }
  },
  "tics": false,
  "stretch": true,
  "hours12": false,
  "live": false
}
