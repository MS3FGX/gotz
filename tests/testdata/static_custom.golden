                                now v 16:00
Local   : Sat 24 Aug 1985 14:00     |
____________OOOOOOXXXXXXXXXXXXXXXXXX|XXXXXXXXXXXOOOOOOOOOOOO____________
New York: Sat 24 Aug 1985 10:00     |
________________________OOOOOOXXXXXX|XXXXXXXXXXXXXXXXXXXXXXXOOOOOOOOOOOO
Berlin  : Sat 24 Aug 1985 16:00     |
______OOOOOOXXXXXXXXXXXXXXXXXXXXXXXX|XXXXXOOOOOOOOOOOO__________________
Shanghai: Sat 24 Aug 1985 22:00     |
XXXXXXXXXXXXXXXXXXXXXXXXOOOOOOOOOOOO|_______________________OOOOOOXXXXXX
Sydney  : Sun 25 Aug 1985 00:00     |
XXXXXXXXXXXXXXXXXXOOOOOOOOOOOO______|_________________OOOOOOXXXXXXXXXXXX
